cookbook/security/form_login_setup.rst

```diff
@@ -165,13 +165,14 @@ form::
 
     // src/AppBundle/Controller/SecurityController.php
     // ...
-    
+
     // ADD THIS use STATEMENT above your class
     use Symfony\Component\Security\Core\SecurityContextInterface;
 
     public function loginAction(Request $request)
     {
         $session = $request->getSession();
+        $error = null;
```

Comment:
Good suggestion, this might not be the way it would usually be done in the Symfony core, but it's actually more readable. Changed it.